lib/meta-boxes.php

```diff
@@ -119,6 +126,20 @@ function igv_cmb_metaboxes() {
   ) );
 
   $obra_meta->add_field( array(
+    'name'       => __( 'Image', 'cmb2' ),
+    'desc'       => __( 'if single image not slideshow', 'cmb2' ),
+    'id'         => $prefix . 'image',
+    'type'       => 'file',
+  ) );
+
+  $obra_meta->add_field( array(
+    'name'       => __( 'Video', 'cmb2' ),
+    'desc'       => __( 'if video not slideshow. [vimeo embed id or ???]', 'cmb2' ),
```

Comment:
im thinking embed code or link
if link, we can convert it to embed code with oembed
either way we can later wrap it with a responsive container

thoughts?